config/opal_functions.m4

```diff
@@ -395,6 +395,10 @@ AC_DEFUN([OPAL_FLAGS_UNIQ],[
                 opal_found=0
                 opal_i=`expr $opal_count + 1`
                 ;;
+        --include)
```

Comment:
Is it `--include` or `-include`?  https://github.com/open-mpi/ompi/issues/9129#issuecomment-876564999 specifically mentions `-include`.  I also see `-include` in https://gcc.gnu.org/onlinedocs/gcc-11.1.0/gcc/Option-Summary.html#Option-Summary.

Reply:
great point, let me double check that

Reply:
it turns out Open MPI is just fine, and only PMIx has to be modified (to support both `--include` and `-include`).
I will try to figure out where the difference in behavior comes from ... later.